spec/support/request/checkout_helper.rb

```diff
@@ -38,16 +38,16 @@ def fill_out_billing_address
     end
   end
 
-  def fill_out_form
-    choose free_shipping.name
-    choose check_without_fee.name
+  def fill_out_form(shipping_method_name, payment_method_name, save_default_addresses = false)
```

Comment:
a keyboard argument for `save_default_addresses` would shed some light on what this flag does when seeing it in use. It saves a jump to the method definition.

Reply:
done :+1: